devops/scripts/create-sycl-user.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# By default Ubuntu sets an arbitrary UID value, that is different from host
+# system. When CI passes default UID value of 1001, some of LLVM tools fail to
+# discover user home directory and fail a few LIT tests. Fixes UID and GID to
+# 1001, that is used as default by GitHub Actions.
+groupadd -g 1001 sycl && useradd sycl -u 1001 -g 1001 -m -s /bin/bash
+# Add sycl user to video/irc groups so that it can access GPU
+usermod -aG video sycl
+usermod -aG irc sycl
+
+# group 109 is required for sycl user to access PVC card.
+groupadd -g 109 render
+usermod -aG render sycl
+
+if [[ -f /run/secrets/sycl_passwd ]]; then
+  # When running in our CI environment, we restrict access to root.
+
+  # Set password for sycl user
+  cat /run/secrets/sycl_passwd | passwd -s sycl
+
+  # Allow sycl user to run as sudo, but only with password
+  echo "sycl  ALL=(root) PASSWD:ALL" >> /etc/sudoers
+else
+  # Otherwise, we allow password-less root to simplify building other
```

Comment:
im not sure if this branch will ever be hit, the only place this script seems get run is inside the Dockerfile creation, so it will only run in CI, not when someone is using the image.

Reply:
> im not sure if this branch will ever be hit, the only place this script seems get run is inside the Dockerfile creation, so it will only run in CI, not when someone is using the image.

Yeah, that's true, that `if` is definitely in correct.

My thinking is that we should have two users: `sycl_ci` which has a password and `sycl` which has password-less root access, but only available in `prebuilt` images that are for end users and not for our CI.

Reply:
Made two users in 0964ce8c4d642cda743e3879982797853b3a16c2